controller/eks-cluster-config-handler.go

```diff
@@ -1132,7 +1133,7 @@ func (h *Handler) importCluster(config *v13.EKSClusterConfig, eksService *eks.EK
 
 // createCASecret creates a secret containing ca and endpoint. These can be used to create a kubeconfig via
 // the go sdk
-func (h *Handler) createCASecret(name, namespace string, clusterState *eks.DescribeClusterOutput) error {
+func (h *Handler) createCASecret(name, namespace string, config *eksv1.EKSClusterConfig, clusterState *eks.DescribeClusterOutput) error {
```

Comment:
Is the `name` and `namespace` provided to this function always `config.Name` or `config.Namespace`? If so, we should remove the `name` and `namespace` from this since it already takes in the `config`.

Reply:
seems like this was updated in the latest commit

Reply:
Yes good catch, fixed.